django_project/base/templates/project/detail.html

```diff
@@ -29,9 +29,20 @@
     {% endfor %}
   {% endif %}
   {% if versions %}
-    <div class="row">
-      <h3>Versions</h3>
-      <hr/>
+          <div class="page-header">
```

Comment:
@rischanlab we should be making atomic PR's. That means one PR for one github issue as fat as possible. It makes it much easier to track the changes related to an issue and also to review changes as they come in.